includes/api-management-multi-region-concepts.md

```diff
@@ -7,12 +7,15 @@ ms.author: danlep
 ---
 
 
-* With multi-region deployment, only the [gateway component](../articles/api-management/api-management-key-concepts.md#api-management-components) of your API Management instance is replicated to multiple regions. The instance's management plane and developer portal remain hosted only in the *primary* region, the region where you originally deployed the service. 
+* With multi-region deployment, only the [gateway component](../articles/api-management/api-management-key-concepts.md#api-management-components) of your API Management instance is replicated to multiple regions. The instance's management plane and developer portal remain hosted only in the *primary* region, the region where you originally deployed the service.
+  
+* If you want to configure a secondary location for your APIM, the VNET and subnet region should match with the secondary location you're configuring. If you're adding, removing, or enabling the Availability zone in the Primary region, or if you're changing the subnet of the primary region, then the VIP of APIM will change. For more information, see [IP addresses of Azure API Management service](/azure/api-management/api-management-howto-ip-addresses#changes-to-the-ip-addresses).
```

Comment:
```suggestion
* If you want to configure a secondary location for your APIM, the VNET and subnet region should match with the secondary location you're configuring. If you're adding, removing, or enabling the Availability zone in the Primary region, or if you're changing the subnet of the primary region, then the VIP of APIM will change. For more information, see [IP addresses of Azure API Management service](/azure/api-management/api-management-howto-ip-addresses#changes-to-the-ip-addresses). However, if you're adding a secondary region, the primary region's VIP of APIM won't change because every region has its own private VIP.
```